apps/passport-server/test/telegram.spec.ts

```diff
@@ -219,4 +241,54 @@ describe("telegram bot functionality", function () {
       expect(insertedEvent?.anon_chat_id).to.eq(anonChannelID_1.toString());
     }
   );
+
+  step(
+    "fetching a telegram chat via a list of eventIds should work",
+    async () => {
+      const sampleChats: ChatIDWithEvents[] = [
+        {
+          telegramChatID: "chat1",
+          ticketEventIds: ["event1", "event2", "event3"]
+        },
+        {
+          telegramChatID: "chat2",
+          ticketEventIds: ["event2", "event3", "event4"]
```

Comment:
event2 and event3 are in both chat 1 and 2. Would an event ever be linked to more than one chat?

Reply:
Good point, updated the test case